src/dataset-builder/DatasetBuilderUtils.test.ts

```diff
@@ -179,3 +181,97 @@ describe('test conversion of DatasetAccessRequest', () => {
     expect(convertDatasetAccessRequest(datasetAccessRequest)).toStrictEqual(datasetAccessRequestApi);
   });
 });
+
+describe('test HighlightConceptName', () => {
+  test('searching beginning of conceptName', () => {
+    const searchFilter = 'Clinic';
```

Comment:
your tests of highlightConceptName look great! I'm wondering if we want a test for ConceptSearch as well since that page has been altered and we may want to show that highlightConceptName appears on the page as expected.